check: true causes run_command (which will error with chroot-git) to fail

reverts part of be1a8e62cfe70af0b0dd97a08271d177d0c2b1d3

--- a/meson.build
+++ b/meson.build
@@ -43,7 +43,7 @@
 package_version = meson.project_version()
 
 if git.found()
-	git_version = run_command('git', ['rev-parse', 'HEAD'], check: true)
+	git_version = run_command('git', ['rev-parse', 'HEAD'])
 	if git_version.returncode() == 0
 		package_version += ' (git-'+git_version.stdout().strip()+')'
 	endif
